spec/pg/decoder_spec.cr

```diff
@@ -58,6 +59,20 @@ describe PG::Decoders do
     x.call("nan").nan?.should be_true
   end
 
+  it "decodes many uuids (#148)" do
+    PG_DB.exec("create table test_uuids(id uuid primary key)")
+    begin
+      1000.times do
+        PG_DB.exec("insert into test_uuids (id) values ($1)", UUID.random.to_s)
+      end
+      ids = PG_DB.query_all("select id from test_uuids limit 909", as: String)
+      id = PG_DB.query_one("select id from test_uuids offset 908 limit 1", as: String)
+      ids.last.should eq(id)
+    ensure
+      PG_DB.exec("drop table test_uuids")
+    end
+  end
```

Comment:
Do you mind changing this something like

``` crystal
  uuid = "aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa"
  ids = db.query_all("select '#{uuid}'::uuid from generate_series(1,1000)", as: String)
  ids.all? { |u| u == uuid }.should eq(true)
```

it's a lot faster and still appears to reproduce the problem
```
generate_series   3.03k (329.67µs) (± 1.13%)  182kB/op         fastest
   create table   6.75  (148.06ms) (± 2.56%)  668kB/op  449.13× slower
```

Reply:
Great idea, done!

I changed the last spec line from:

```crystal
ids.all? { |u| u == uuid }.should eq(true)
```

to

```crystal
ids.uniq.should eq([uuid])
```

which gives a more helpful message when it fails (you can see the wrong uuid there).

Reply:
(but let me know if you prefer the other one)

Reply:
No, that's great, I just went with the first assertion I thought of